internal/rbd/rbd_util.go

```diff
@@ -2011,3 +2015,25 @@ func getCephClientLogFileName(id, logDir, prefix string) string {
 
 	return fmt.Sprintf("%s/%s-%s.log", logDir, prefix, id)
 }
+
+// strategicActionOnLogFile act on log file based on cephLogStrategy.
+func strategicActionOnLogFile(logStrategy, logFile string) error {
+	var err error
+
+	switch strings.ToLower(logStrategy) {
+	case "compress":
+		if err = log.GzipLogFile(logFile); err != nil {
```

Comment:
@Madhu-1 I tried changing this to:
```
go func(logFile string) {                                                                     
        var err error                                                                         
        if err = log.GzipLogFile(logFile); err != nil {                                       
                log.WarningLog(ctx, "failed to compress logfile: %s, error: %v", logFile, err)
        }                                                                                     
}(logFile)                                                                                    
```

But it didn't work! An immediate check for .gz file after calling `strategicActionOnLogFile()` in the unit tests `TestStrategicActionOnLogFile()` return err :-(

Reply:
May be we can check with some timeout or with forloop?

Reply:
Fixed it now.

Reply:
Fixed it now.